lib/manual-remediation.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
```

Comment:
What is this file? I'm guessing its some kind of manual test file; would prefer if this was in `test/` or `scripts/` to distinguish between this and production code, if possible

Reply:
Yep, dropping it.